src/main/c/cext/ruby.c

```diff
@@ -1202,6 +1202,7 @@ VALUE rb_str_buf_cat(VALUE string, const char *to_concat, long length) {
   return rb_str_cat(string, to_concat, length);
 }
 
+// returns Truffle::CExts::RbEncoding, takes Encoding or String
```

Comment:
It's Truffle::CExt, but I would simplify to just `RbEncoding` here for readability.

Reply:
Fixing the typo. I am keeping it explicit though, this is a C file there is nowhere implied that it's in Truffle::CExt scope.

Reply:
The typo is still there.